### ISSUE TITLE
Cannot read properties of undefined (reading 'componentName')

### ISSUE THREAD

**Issue by BramEsposito** [windx-foobar/nuxt3-notifications#5]
When cloning this repository and running `npm run dev`, I am greeted with this error in the browser:

```
Cannot read properties of undefined (reading 'componentName')

at ./node_modules/nuxt/dist/app/nuxt.js:144:60
at fn (./node_modules/nuxt/dist/app/nuxt.js:226:44)
at Object.callAsync (./node_modules/unctx/dist/index.mjs:68:55)
at ./node_modules/nuxt/dist/app/nuxt.js:229:56
at Object.runWithContext (./node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:3972:18)
at callWithNuxt (./node_modules/nuxt/dist/app/nuxt.js:229:24)
at ./node_modules/nuxt/dist/app/nuxt.js:42:41
at EffectScope.run (./node_modules/@vue/reactivity/dist/reactivity.cjs.js:47:16)
at Object.runWithContext (./node_modules/nuxt/dist/app/nuxt.js:42:31)
```

This also happens in my own project since upgrading Nuxt to the latest release (V3.12.1)

npm: 10.8.1
node: 22.0.0

**Comment by BramEsposito** [comment]
@windx-foobar can you please review and merge this change? Thanks a lot

**Comment by windx-foobar** [comment]
Hi!
Thank you very match.
Can u check the changes I made? #6

Reason:
> We cannot use the updateRuntimeConfig utility directly, because This is a late feature from Nuxt. In the package dependencies you can see the declared support for nuxt@^3.0.0 and higher, and this appeared relatively recently in nuxt@^3.11.0. Sorry for my broken English:)

@BramEsposito

**Comment by BramEsposito** [comment]
Good catch! 

Didn't think to check compatibility with older Nuxt versions. Your changes work fine in my v3.12.1 Nuxt project so you. have my full support.

Thanks a lot for making this project and your swift reply! 💯

**Comment by windx-foobar** [comment]
> Good catch!
> 
> Didn't think to check compatibility with older Nuxt versions. Your changes work fine in my v3.12.1 Nuxt project so you. have my full support.
> 
> Thanks a lot for making this project and your swift reply! 💯

Great. I merge it and release next minor version. Thank u very much 🚀
Can u answer please this fix decides it #1 ?

**Comment by windx-foobar** [comment]
resolved #6
